magma/uniquification.py

```diff
@@ -60,6 +56,8 @@ def _rename(self, ckt, new_name):
         type(ckt).rename(ckt, new_name)
 
     def __call__(self, definition):
+        for module in definition.bind_modules:
```

Comment:
@leonardt I think this is a better way to do it than the below removed statements (L72-73, L81-83).